feature/policy_forwarding/encapsulation/otg_tests/match_ip_set_ttl_ipv4/README.md

```diff
@@ -0,0 +1,197 @@
+# PF-1.11 Policy forwarding rule to match IP and rewrite TTL
+
+This test uses policy-forwarding to set the IP TTL.  It contains 2 scenarios as subtests:
+1. Apply this policy alone on an ingress interface.
+2. Apply this policy in combination with a gRIBI programmed next-hop which performs encapsulation and sets the TTL on the outer, encapsulation packet.
+
+## Topology
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Test setup
+
+TODO: Complete test environment setup steps
+
+inner_ipv6_dst_A = "2001:aa:bb::1/128"
+inner_ipv6_dst_B = "2001:aa:bb::2/128"
+inner_ipv6_default = "::/0"
+
+ipv4_inner_dst_A = "10.5.1.1/32"
+ipv4_inner_dst_B = "10.5.1.2/32"
+ipv4_inner_default = "0.0.0.0/0"
+
+outer_ipv6_src =      "2001:f:a:1::0"
+outer_ipv6_dst_A =    "2001:f:c:e::1"
+outer_ipv6_dst_B =    "2001:f:c:e::2"
+outer_ipv6_dst_def =  "2001:1:1:1::0"
+outer_dst_udp_port =  "6635"
+outer_dscp =          "26"
+outer_ip-ttl =        "64"
+
+## Procedure
+
+### TE-1.11.1 Rewrite the packet TTL = 1 if matching a specified destination IP.
+**[TODO]** Test code needs to be implemented.
+
+Canonical OpenConfig for policy forwarding, matching IP prefix and TTL = 1 with action
+set packet TTL = 1.
+
+```json
+{
+  "openconfig-network-instance": {
+    "network-instances": [
+      {
+        "afts": {
+          "policy-forwarding": {
+            "policies": [
+              {
+                "config": {
+                  "policy-id": "retain ttl",
+                  "type": "PBR_POLICY"
+                },
+                "policy": "retain ttl",
+                "rules": [
+                  {
+                    "config": {
+                      "sequence-id": 1,
+                    },
+                    "ipv6": {
+                      "config": {
+                        "destination-address": "router_ip"
```

Comment:
What is `router_ip` here? It doesn't seem to be specified. Can we make this the concrete IP address based on the assignments made above?